### DISCUSSION TITLE
Inline Search Linking

### DISCUSSION THREAD

**Discussion by mikewolfd** [outline/outline#5360] [Ideas]
I want a shortcut to make a specific word(s) into a searchable link, allowing more of a knowledge-base style of behavior. 

Example: Make the word "React" a link that would send you to the search page with prepopulated results.

**Comment by tommoor** [comment]
Is hashtags more like what you're after here? Of course in the meantime you can create a link to the search page – the url won't be changing any time.

**Reply by mikewolfd** [reply]
Yes, that's exactly what I'm thinking, fair enough. Is tagging on the feature roadmap?
